.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow opens and updates a pull request with a new package version
+# based on code changes. Merging the pull request updates the version in
+# composer.json, updates the changelog and creates release tags.
```

Comment:
Sorry for nit-picking, but is there any specific reason why we've deviated from the format introduced in [buttercms-js](https://github.com/ButterCMS/buttercms-js/blob/master/.github/workflows/release-please.yml) and [buttercms-ruby](https://github.com/ButterCMS/buttercms-ruby/blob/237400f9d7957c6da93e20193dd8c03885462af3/.github/workflows/release-please.yml) (
```diff
- based on code changes.
- 
- The pull request
+ based on code changes. Merging the pull request
```
)? 😅

Reply:
I guess I felt that in this case, when there is an additional action to describe, the newlines corresponded better this way. And while I was at it, I added the word "merging" because it is a slightly more exact description of what is going on. Do you want me to unify the description across all repos @Toreno96 ?

Reply:
> when there is an additional action to describe, the newlines corresponded better this way

That makes sense.

> Do you want me to unify the description across all repos

Nah, consistency would be nice, but that would be too much hassle for such a minor improvement, I think. Maybe in the future if you happen to change something else in those repos?